pkg/app/app.go

```diff
@@ -75,6 +78,33 @@ func RealMain(opts types.Options, d types.Deployer, tester types.Tester) (result
 		return errors.Wrap(err, "could not create runner output")
 	}
 	writer := metadata.NewWriter("kubetest2", junitRunner)
+
+	var wg sync.WaitGroup
+	wg.Add(1)
```

Comment:
Do we need a wait group?

In most happy paths this signal handling shouldn't be required and won't be triggered.
We already have the done channel, so we probably don't need an additional wait group.

Reply:
Agree with you. We do not need waitgroup since we have done channel  🤦‍♀️